gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/ImportCRDUseCase.java

```diff
@@ -170,6 +174,8 @@ private ApiCRDStatus update(Input input, Api existingApi) {
         try {
             var updated = updateApiDomainService.update(existingApi.getId(), input.crd, input.auditInfo);
```

Comment:
I would put the `updateApiCategoriesToIds` in this method --> `UpdateApiDomainService.update` since we want it to happen for every update.

Reply:
Done